src/amo/components/ReportAbuseButton/styles.scss

```diff
@@ -0,0 +1,53 @@
+@import "~ui/css/vars";
+
+.ReportAbuseButton {
+  margin: 10px auto;
+}
+
+.ReportAbuseButton-header {
+  margin-bottom: 0;
+}
+
+.ReportAbuseButton-first-paragraph {
+  margin-top: 6px;
+}
+
+.ReportAbuseButton--preview {
+  display: block;
+
+  .ReportAbuseButton--is-expanded & {
+    display: none;
+  }
+}
+
+.ReportAbuseButton--expanded {
+  display: none;
+
+  .ReportAbuseButton--is-expanded & {
+    display: block;
+  }
+}
+
+.ReportAbuseButton-textarea {
+  font-size: $font-size-m-smaller;
+  line-height: 1.4;
+  margin: 5px auto;
+  min-height: 51px;
```

Comment:
It's how tall it is with two lines. Because the text-area grows/shrinks with input, it's how tall the input will be when there is two lines of placeholder text (as there often is). So it makes the transition when you start typing less jarring by not shrinking the size of the textarea.

I'll add a comment 😄